erpnext/selling/doctype/customer/customer.py

```diff
@@ -20,6 +20,7 @@ def onload(self):
 		"""Load address and contacts in `__onload`"""
 		load_address_and_contact(self)
 		self.load_dashboard_info()
+		self.fetch_primary_address_and_contact_detail()
```

Comment:
Why this method? If user want to update the value of mobile no, email id  then he will reselect the contact in the customer form

Reply:
Exactly. The User has to know about it or there must be some way by which the form gets dirty and user comes to know that he/she has to reselect the option. Just a thought if something can be done for this or I'll revert the changes but at some later point one might consider it as a bug.